packages/react/README.md

```diff
@@ -40,7 +40,7 @@ Please note that Castor is exported as an [ECMAScript module](https://developer.
 
 You may wish to configure your bundler to transpile to a different module syntax, and/or to target lower a ECMAScript version.
 
-For example, you might choose UMD module syntax targeting ES5 if your app needs to support IE11.
+For example, you might choose UMD module syntax targeting ES5 if your app needs to support IE11 (please note that we do not support old browsers - use at your own risk).
```

Comment:
I don't like the `use at your own risk` part, feels like saying there is in fact an inherent risk.

Also, `old` is subjective, what browsers do we support?
Should we have a dedicated section to list them?

Reply:
So I mentioned this in the description, that we actually _do not_ have a list of supported browsers.

Something to think about? Probably we should.

What do you think about just "please note that we do not test Castor against IE11 or previous versions".

Reply:
Yea we probably should, and create a `.browserslist` file too.

That's fine, or `please note that Castor is not tested in Internet Explorer`.

Reply:
https://github.com/onfido/castor/issues/243